test/jit/test_peephole.py

```diff
@@ -518,6 +518,15 @@ def foo2(x: List[int]):
             FileCheck().check_not("aten::eq").check_not("aten::neq").run(func_s.graph)
             self.assertEqual(func(inp), func_s(inp))
 
+    def test_peephole_add_zero(self):
+        @torch.jit.script
+        def foo (x: int):
+            return x + 0, 0 + x
+
+        self.run_pass("peephole", foo.graph)
+        FileCheck().check_not("aten::add")
```

Comment:
`.run(foo.graph)`

Reply:
nice, i actually tried to make FileCheck throw if it hasn't been run but i think you can't like throw an exception in C++ in the destructor, or something weird happens..